global.R

```diff
@@ -32,4 +32,17 @@ data_master_raw <-
                             sheet = "Master",
                             na = c("N/A", "")) %>% 
   # infoBox colour indicating if FB link exists
-  mutate(exists_fb = if_else(condition = !is.na(x = facebook), "blue", "black"))
\ No newline at end of file
+  mutate(exists_fb = if_else(condition = !is.na(x = facebook), "blue", "black"))
+
+# Version to be shown with DT on app
+data_master_details <-
+  googlesheets4::read_sheet(ss = sheet_url,
+                            sheet = "Master",
+                            na = c("N/A", "")) %>%
+  select(Constituency = "DropDownText",
+         Party,
+         DC,
+         Region,
+         District,
+         FB = "facebook")
```

Comment:
I'd suggest we remove this code as it's doing the same stuff as `data_master_raw`, just extracting specific columns and renaming the `DropDownText` and `facebook` ones.

A better performing approach would do the renaming in the `server.R` file where you call `data_master_details` (Line 89). Here, you can do something like:

> `renderDT(
>       `select(.data = data_master_details,`
>                   `Constituency = DropDownText,`
>                   `Party,`
>                   `DC,`
>                   `Region,`
>                   `District,`
>                   `FB = Facebook)`

Reply:
Makes total sense, and agreed - now implemented